newsletter-template.md

```diff
@@ -1,5 +1,5 @@
 +++
-title = "This Month in Rust GameDev #{TODO} - {TODO} {TODO}"
+title = "This Month in Rust GameDev #{{ NEWSLETTER_COUNTER }} - {{ NEWSLETTER_MONTH }} {{ NEWSLETTER_YEAR }}"
 transparent = true
 date = TODO
```

Comment:
Should this be templated too?

Reply:
I left this blank because it's supposed to be the date the post is actually published. I could fill in the deadline if you want (3rd of the next month).

Reply:
Normally we set a date that would be nice to have it done by (say the 1st if we really are ahead of timelines), and update once it does actually get published. Also, it fails building the site with just TODO 😅 

![image](https://github.com/rust-gamedev/rust-gamedev.github.io/assets/14791619/3eed1afd-360f-46ef-9c6b-3c2275ea7247)

Reply:
There you go: [`f1b2a80` (#1464)](https://github.com/rust-gamedev/rust-gamedev.github.io/pull/1464/commits/f1b2a8026f06a3a2b15ce50e775a0e005d42c019)